zenoh-router/Cargo.toml

```diff
@@ -33,7 +33,7 @@ maintenance = { status = "actively-developed" }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-async-std = { version = "=1.8.0", features = ["unstable"] }
+async-std = { version = "=1.9.0", features = ["unstable"] }
```

Comment:
features = ["unstable"] was needed until async-std 1.8.0 to support the async channels. 
Since the channels have been stabilised in async-std 1.9.0, features = ["unstable"] should be hence removed.